lib/tmuxinator/cli.rb

```diff
@@ -48,64 +56,93 @@ def completions(arg)
     map "o" => :new
     map "e" => :new
     map "n" => :new
+    method_option :local, type: :boolean,
+                          aliases: ["-l"],
+                          desc: "Create local project file at ./.tmuxinator.yml"
 
     def new(name)
-      config = Tmuxinator::Config.project(name)
-
-      unless Tmuxinator::Config.exists?(name)
-        template = Tmuxinator::Config.default? ? Tmuxinator::Config.default : Tmuxinator::Config.sample
-        erb  = Erubis::Eruby.new(File.read(template)).result(binding)
-        File.open(config, "w") { |f| f.write(erb) }
+      project_file = if options[:local]
+                       Tmuxinator::Config::LOCAL_DEFAULT
+                     else
+                       Tmuxinator::Config.default_project(name)
+                     end
+      unless File.exists?(project_file)
```

Comment:
I was unable to get this to fail when running locally.  Just to be sure, I put in a couple more tests that specifically check for the different project file paths (local or traditional).  Those are passing (locally, natch), as well.

I don't think this would mean that other `#exists?` checks would need to be changed.  My reading is that `Tmuxinator::Config.exists?` and `Tmuxinator::Config.local?` are intended to be used for checking for those specific conditions; perhaps `exists?` should be renamed `traditional?` or something like that to delineate it from `local?`.  Regardless, the `File.exists?` check here is literally a filesystem check that the selected project file path does or does not already exist, so I don't think it'd need to be changed.